src/main/java/org/gridsuite/shortcircuit/server/ShortCircuitController.java

```diff
@@ -97,18 +94,31 @@ public ResponseEntity<ShortCircuitAnalysisResult> getResult(@Parameter(descripti
                 : ResponseEntity.notFound().build();
     }
 
-    @GetMapping(value = "/results/{resultUuid}/fault_results/paged", produces = APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/results/{resultUuid}/fault_results", produces = APPLICATION_JSON_VALUE)
     @Operation(summary = "Get a fault results page for a given short circuit analysis result")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The page of fault results"),
         @ApiResponse(responseCode = "404", description = "Short circuit analysis result has not been found")})
     public ResponseEntity<Page<FaultResult>> getPagedFaultResults(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid,
-                                                                @Parameter(description = "Full or only those with limit violations or none fault results") @RequestParam(name = "mode", required = false, defaultValue = "WITH_LIMIT_VIOLATIONS") FaultResultsMode mode,
-                                                                Pageable pageable) {
+                                                                  @Parameter(description = "Full or only those with limit violations or none fault results") @RequestParam(name = "mode", required = false, defaultValue = "WITH_LIMIT_VIOLATIONS") FaultResultsMode mode,
```

Comment:
update mode description

Reply:
Can I give a link to the JavaDoc of the enum ? Because it can be quite long and the same.